src/shared.js

```diff
@@ -863,6 +863,30 @@ export const searchSpecimenInstitute = async() => {
     }
 }
 
+/**
+ * Fetches biospecimen collection data from the database via login site number 
+ * @param {number} login site number
+ * @returns {object|array} returns a response object if response is 200 or an empty array
+ * 
+ */
+export const searchSpecimenByRequestedSite = async (requestedSite) => {
+    const idToken = await getIdToken();
+    const response = await fetch(`${api}api=searchSpecimen&requestedSite=${requestedSite}`, {
+    method: "GET",
+    headers: {
+        Authorization:"Bearer "+idToken
+        }
+    });
+    if (response.status === 200) {
+        const responseObject = await response.json();
+        return responseObject;
+    }
+    else {
+        console.error("getSpecimensByRequestedSite's responseObject status code not 200!");
+        return [];
```

Comment:
```suggestion
        return {data:[]};
```
to make returned data type consistent.